skrub/datasets/_utils.py

```diff
@@ -11,13 +41,14 @@ def get_data_dir(name: str | None = None) -> Path:
 
     Parameters
     ----------
+    data_home : Path, default=None
```

Comment:
Can you order the parameter as in the function signature?

Reply:
I would also support an `str` or `Path`.

Reply:
:+1: